python/requirements/ml/requirements_dl.txt

```diff
@@ -5,9 +5,7 @@
 # If you make changes to this file, please also change `requirements_ml_docker.txt`!
 
 
-# TODO(amogkam): Remove after https://github.com/tensorflow/tensorflow/issues/52922 is fixed.
-keras==2.6.0
-tensorflow==2.6.0
+tensorflow~=2.6.2
```

Comment:
maybe remove here and only have it in base `requirements.txt`?

Reply:
Anything against allowing tensorflow 2.7 or 2.8 btw? Probably one of the [breaking changes](https://github.com/tensorflow/tensorflow/releases/tag/v2.7.0) in 2.7?

Reply:
Usually it's a breaking change but cc @sven1977 for more details

Reply:
Looks like this sync up is necessary indeed. There's a note in their [docs](https://www.tensorflow.org/probability/install):
> Note: Since TensorFlow is not included as a dependency of the TensorFlow Probability package (in `setup.py`), you must explicitly install the TensorFlow package (`tensorflow` or `tensorflow-gpu`). This allows us to maintain one package instead of separate packages for CPU and GPU-enabled TensorFlow.

Which is actually outdated as `tensorflow-gpu` is deprecated. They could simply have the correct tensorflow requirement range specified in the probability package requirements, and there would be no need to manually sync up here anymore (i.e. simply only specify `tensorflow-probability==XXX` in `python/requirements.txt`).